pkgs/applications/editors/emacs/elisp-packages/elpa-generated.nix

```diff
@@ -3915,10 +4009,10 @@
       elpaBuild {
         pname = "org";
         ename = "org";
-        version = "9.6.19";
+        version = "9.6.22";
         src = fetchurl {
-          url = "https://elpa.gnu.org/packages/org-9.6.19.tar";
-          sha256 = "0ibgw0i7nsn589k0ynifwdp1f3ia6p8369myhjqgmwy392cwrcxg";
+          url = "https://elpa.gnu.org/packages/org-9.6.22.tar";
```

Comment:
Org mode just [releases][1] 9.6.23 to fix several critical vulnerabilities. The release is coordinated with emergency Emacs 29.3 release.

That version is in emacs-overlay now.  Could you update this PR to include that?

[1]: https://list.orgmode.org/87o7b3eczr.fsf@bzg.fr/T/#t

Reply:
I guess we should also update Org mode to 9.6.23 for NixOS 23.11.

Reply:
Yes.

Reply:
> I guess we should also update Org mode to 9.6.23 for NixOS 23.11.

https://github.com/NixOS/nixpkgs/pull/298993